spec/jobs/ingest_local_file_job_spec.rb

```diff
@@ -1,33 +1,26 @@
 require 'spec_helper'
 
-### TODO - Needs Refactoring after Sufia is Broken up
 describe IngestLocalFileJob do
-  # let(:user) { FactoryGirl.find_or_create(:jill) }
-  #
-  # let(:generic_file) { create :generic_file, depositor: user }
-  #
-  # let(:job) { IngestLocalFileJob.new(generic_file.id, mock_upload_directory, "world.png", user.user_key) }
-  # let(:mock_upload_directory) { 'spec/mock_upload_directory' }
-  #
-  # before do
-  #   Dir.mkdir mock_upload_directory unless File.exists? mock_upload_directory
-  #   FileUtils.copy(File.expand_path('../../fixtures/world.png', __FILE__), mock_upload_directory)
-  # end
-  #
-  # it "should have attached a file" do
-  #   job.run
-  #   expect(generic_file.reload.content.size).to eq(4218)
-  # end
-  #
-  # describe "virus checking" do
```

Comment:
Virus checking is added as a validation in [generic_file_behavior](https://github.com/projecthydra-labs/curation_concerns/blob/master/curation_concerns-models/app/models/concerns/curation_concerns/generic_file_behavior.rb#L14). There is [test coverage](https://github.com/projecthydra-labs/curation_concerns/blob/master/spec/models/curation_concerns/generic_file_spec.rb#L358) for that so testing this here is redundant.